plugins/Microsoft.DocAsCode.Build.JavaScriptReference/resources/partials/class.header.tmpl.partial

```diff
@@ -0,0 +1,43 @@
+{{!Copyright (c) Microsoft. All rights reserved. Licensed under the MIT license. See LICENSE file in the project root for full license information.}}
+
+<h1 id="{{id}}" data-uid="{{uid}}">{{>partials/title}}</h1>
+<div class="markdown level0 summary">{{{summary}}}</div>
+<div class="markdown level0 conceptual">{{{conceptual}}}</div>
+{{#inheritance.0}}
+<div class="inheritance">
+  <h5>{{__global.inheritance}}</h5>
+{{/inheritance.0}}
```

Comment:
>inheritance [](start = 3, length = 11)

please also help to update src\docfx.website.themes\default\partials\class.header.tmpl.partial